templates/search/blocks/search_filters.html

```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
                     <div class="search-filters__form-block">
                         <h3 class="tna-heading sr-only">Edit filters</h3>
 
-                        {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
+                        {% if form.group.value == 'tna' or form.group.value == 'digitised' %}
```

Comment:
```suggestion
                        {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
```
This condition of for Covering dates/Dates. No change required. Restore it.

Reply:
restoring values.